### PR TITLE
Add 2D & 3D navigation minimum setup tutorials

### PR DESCRIPTION
**Pull request by smix8**
Adds illustrated minimum setup tutorials for navigation in 2D and 3D using NavigationAgent nodes with script example.

### CODE REVIEW

File: tutorials/navigation/navigation_setup_2d_scene.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.. _doc_navigation_setup_2d_scene:
+
+Setup for 2D scene
+==========================================
```
Comment (Calinou) [inline tutorials/navigation/navigation_setup_2d_scene.rst:4]: Number of `=` should match the title (this is a reStructuredText requirement):

```suggestion
Setup for 2D scene
==================
```

Same for `-` and `^` headings.

File: tutorials/navigation/navigation_advanced.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.. _doc_navigation_advanced:
```
Comment (Calinou) [inline tutorials/navigation/navigation_advanced.rst:1]: Did you mean to add this page now?

File: tutorials/navigation/navigation_expert.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.. _doc_navigation_expert:
```
Comment (Calinou) [inline tutorials/navigation/navigation_expert.rst:1]: Same as above.

File: tutorials/navigation/navigation_advanced.rst (hunk 2 above)
Comment (smix8) [reply tutorials/navigation/navigation_advanced.rst:1]: No I didn't, this is a leftover from an old iteration of the complete docs.

File: tutorials/navigation/navigation_expert.rst (hunk 3 above)
Comment (smix8) [reply tutorials/navigation/navigation_expert.rst:1]: Same

**Review by mhilbrunner — APPROVED** [review]

**Review by skyace65 — COMMENTED** [review]
I don't like the idea of having a different page going over navigation setup when that's already done on the 3D navigation page. The instructions on that page could definitely use some improvement. I made them with very basic knowledge of how the system works. But there shouldn't be duplicated information across pages.

**Comment by mhilbrunner** [comment]
Closing as superseded by https://github.com/godotengine/godot-docs/pull/6169.

**Pull request closed without merging**